pkg/cli/webhook/github_test.go

```diff
@@ -46,7 +46,7 @@ func TestAskGHWebhookConfig(t *testing.T) {
 				as.StubOne("webhook-secret")
 				as.StubOne("token")
 			},
-			repoURL:       "https:/github.com/pac/demo",
+			repoURL:       "https://github.com/pac/demo",
```

Comment:
was this test a bad url on purpose? because we know validate the URL properly in formatting.GetRepoOwnerURL and it was failing, i wasn't sure of the test.

Reply:
wasn't on purpose 🤔 it didn't failed but